orchestrator/redisClientWrapper.js

```diff
@@ -0,0 +1,78 @@
+/* jslint node: true */
+"use strict";
+var axios = require("axios");
+
+var config = require('./config');
+var auth = require('./auth');
+
+/**
+ * Client for REDIS database
+ */
+class ClientWrapper {
+
+  constructor(client) {
+    this.client = client;
+  }
+  /**
+   * Returns template list of a device. Search for it on cache, if it's not, requests device manager
+   * and then stores on cache.
+   * @param deviceid 
+   */
+
+  getTemplateList(deviceid, tenant) {
+    return new Promise((resolve, reject) => {
+      this.client.get(deviceid + ":" + tenant, (error, data) => {
+        if (error) {
```

Comment:
hum... if we cannot access the cache for some reason, should we ignore it and access the device manager directly?
@giovannicuriel what do you think about it?

Reply:
I guess we have to ignore it and access device-manager directly